src/gtimelog/tests/modules/test_email.py

```diff
@@ -1,23 +1,24 @@
-# -*- coding: utf-8 -*-
-"""Tests for gtimelog.main"""
-
+import doctest
 import textwrap
 import unittest
 
-from unittest import mock
-
+from gtimelog import __version__
+from gtimelog.core.utils import prepare_message
+from gtimelog.tests.commons import Checker
 
-gi = mock.MagicMock()
+gi = unittest.mock.MagicMock()
```

Comment:
Let's not.  I expect to discover any day now that unittest.mock, being an older version of mock, has some bug or lacks some feature that will make me want to switch to using mock from PyPI instead.  I'd rather change one import at the top of each test file than every invocation of unittest.mock then.

Reply:
Fixed